### ISSUE TITLE
Dataset Viewer issue for eli5

### ISSUE THREAD

**Issue by yhifny** [huggingface/datasets-server#1110]
### Link

https://huggingface.co/datasets/eli5

### Description

while calling this script "curl -X GET \
     "https://datasets-server.huggingface.co/first-rows?dataset=eli5&config=LFQA_reddit&split=test_eli5""

I got this error "{"error":"list.extend() takes exactly one argument (7 given)"}"

**Comment by albertvillanova** [comment]
This fix:
- #1116

has changed the error message to the real one. Now: https://datasets-server.huggingface.co/first-rows?dataset=eli5&config=LFQA_reddit&split=test_eli5
gives error:
```
zstd decompress error: Frame requires too much memory for decoding
```

Related to: https://huggingface.co/datasets/eli5/discussions/2
